java-ast/addon/src/main/java/org/jboss/windup/ast/java/JavaASTProcessor.java

```diff
@@ -200,42 +204,59 @@ private JavaASTProcessor(WildcardImportResolver importResolver, CompilationUnit
         cu.accept(this);
     }
 
+    private String extractDefinitionLine(String typeDeclaration)
+    {
+        String typeLine = "";
+        String[] lines = typeDeclaration.split("\n");
+        for (String line : lines)
+        {
+            typeLine = typeLine + "\n" + line;
+            if (line.contains("{"))
+            {
+                break;
+            }
+        }
+        return typeLine;
+    }
```

Comment:
Well, it could span quite a few lines in the raw text. But I think accounting for this with a regex would be painful, and the extra lines are not syntactically significant, IMO.